redis-time-series.gemspec

```diff
@@ -32,6 +32,9 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'redis', '>= 3.3'
+  spec.add_dependency 'connection_pool', '>= 2.5'
+  spec.add_dependency 'tzinfo', '>= 2.0.6'
+  spec.add_dependency 'tzinfo-data', '>= 1.2023.3'
```

Comment:
These are to make the samples wrapper time zone aware? At first blush I don't love the added dependencies, but then again it is Redis _time_ series, so having time-related dependencies isn't beyond the pale 🤔